src/js/bandwidth_test.js

```diff
@@ -205,25 +237,46 @@ VideoBandwidthTest.prototype = {
     this.call.close();
     this.call = null;
 
-    // Checking if greater than 2 because Chrome sometimes reports 2x2 when
-    // a camera starts but fails to deliver frames.
-    if (this.videoStats[0] < 2 && this.videoStats[1] < 2) {
-      this.test.reportError('Camera failure: ' + this.videoStats[0] + 'x' +
-          this.videoStats[1] + '. Cannot test bandwidth without a working ' +
-          ' camera.');
-    } else {
-      this.test.reportSuccess('Video resolution: ' + this.videoStats[0] + 'x' +
-          this.videoStats[1]);
+    // TODO: Remove browser specific stats gathering hack once adapter.js or
+    // browsers converge on a standard.
+    if (webrtcDetectedBrowser === 'chrome') {
+      // Checking if greater than 2 because Chrome sometimes reports 2x2 when
+      // a camera starts but fails to deliver frames.
+      if (this.videoStats[0] < 2 && this.videoStats[1] < 2) {
+        this.test.reportError('Camera failure: ' + this.videoStats[0] + 'x' +
+            this.videoStats[1] + '. Cannot test bandwidth without a working ' +
+            ' camera.');
+      } else {
+        this.test.reportSuccess('Video resolution: ' + this.videoStats[0] +
+            'x' + this.videoStats[1]);
+        this.test.reportInfo('RTT average: ' + this.rttStats.getAverage() +
```

Comment:
There is a lot of duplication between the chrome and firefox conditions.... can you clear it?